src/signing_commitment.rs

```diff
@@ -0,0 +1,102 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+use std::io;
+
+use reddsa::frost::redjubjub::round1::NonceCommitment;
+
+use crate::participant::{Identity, IDENTITY_LEN};
+
+const SIGNING_COMMITMENT_LENGTH: usize = IDENTITY_LEN + 96;
```

Comment:
why + 96? from `deserialize_from` it looks like the hiding and binding commitments are each 32 bytes

Reply:
changed this to reflect the correct number. I believe it is an error introduced here https://github.com/iron-fish/ironfish/pull/4744/files